test/SourceKit/CursorInfo/issue-62457-symbol-graph-crash.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %sourcekitd-test -req=cursor -req-opts=retrieve_symbol_graph=1 -pos=8:13 %s -- %s
```

Comment:
Would it be useful to check the response here as well?

Also, not required, but I find it easier to follow if the `RUN` line is right above the place where you invoke `sourcekitd` test and then uses `-pos=%(line + 1):13`. That also makes it easier to edit the file because adding new lines above the cursor info position doesn’t shift everything around.

Reply:
Makes sense to me 👍🏽